module/validate.go

```diff
@@ -40,14 +41,28 @@ func (m *Module) Execute(targets map[string]pgs.File, pkgs map[string]pgs.Packag
 		}
 
 		for _, tpl := range tpls {
-			out := m.ctx.OutputPath(f)
-			out = out.SetExt(".validate." + tpl.Name())
-			m.AddGeneratorTemplateFile(out.String(), tpl, f)
+			out := templates.FilePathFor(tpl)(f, m.ctx, tpl)
+			if out != nil {
```

Comment:
Probably worth a comment about when out might be nil.

Reply:
Commented.